common-crafting.lic

```diff
@@ -112,6 +112,7 @@ module DRCC
       enchanting_rooms = get_data('crafting')['artificing'][hometown]['brazier-rooms']
       idle_room = get_data('crafting')['artificing'][hometown]['idle-room']
       DRCT.find_sorted_empty_room(enchanting_rooms, idle_room)
+      clean_brazier?
```

Comment:
I *think* it should be OK, but will try to get some testing done this week with own brazier.

Reply:
After running through the test and looking at the logic here, it (correctly) shouldn't.  The reason for checking for a clean brazier here is it's using public/society ones, which someone else could have left with unfinished work on.  If an override room is defined it's because you have your own brazier, so that shouldn't be the case.